src/JBDeadline.sol

```diff
@@ -12,7 +11,7 @@ import {JBRuleset} from "./structs/JBRuleset.sol";
 /// seconds before the current ruleset ends. In other words, rulesets must be queued before the deadline to take effect.
 /// @dev Project rulesets are stored in a queue. Rulesets take effect after the previous ruleset in the queue ends, and
 /// only if they are approved by the previous ruleset's approval hook.
-contract JBDeadline is ERC165, IJBRulesetApprovalHook {
+contract JBDeadline is IJBRulesetApprovalHook {
```

Comment:
is IERC165?

Reply:
rulesetapprovalhook already takes that on